cobalt/browser/h5vcc_accessibility/h5vcc_accessibility_impl.cc

```diff
@@ -0,0 +1,44 @@
+// Copyright 2025 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "cobalt/browser/h5vcc_accessibility/h5vcc_accessibility_impl.h"
+
+#include "base/functional/bind.h"
+#include "base/functional/callback.h"
+#include "build/build_config.h"
+
+namespace h5vcc_accessibility {
+
+H5vccAccessibilityImpl::H5vccAccessibilityImpl(
+    content::RenderFrameHost& render_frame_host,
+    mojo::PendingReceiver<mojom::H5vccAccessibility> receiver)
+    : content::DocumentService<mojom::H5vccAccessibility>(render_frame_host,
+                                                          std::move(receiver)) {
+}
+
+void H5vccAccessibilityImpl::Create(
+    content::RenderFrameHost* render_frame_host,
+    mojo::PendingReceiver<mojom::H5vccAccessibility> receiver) {
+  new H5vccAccessibilityImpl(*render_frame_host, std::move(receiver));
+}
+
+void H5vccAccessibilityImpl::GetTextToSpeech(GetTextToSpeechCallback callback) {
+  // TODO(b/391708407): invoke
+  // starboard/android/shared/accessibility_get_text_to_speech_settings.cc
+  // to return enabled variable.
+  bool enabled = false;
```

Comment:
Call NOTIMPLEMENTED here until it is
If this is going to be a fast follow and you don't want for rebuilds then I'm fine with getting it in without, only because this is high prio